### PR TITLE
[zh] Sync 1.21 release note #14634 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync 1.21 release note #14634 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/releases/1.21.x/announcing-1.21/_index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+aliases:
+- /zh/news/announcing-1.21
+- /zh/news/announcing-1.21.0
+---
+
+我们很高兴地宣布 Istio 1.21 发布。这是 2024 年的第一个 Istio 版本。
+我们要感谢整个 Istio 社区对 1.21.0 版本发布所作出的帮助。
+我们要感谢此版本的几位发布经理，来自 Google 的 `Aryan Gupta`、
+来自 Tetrate 的 `Jianpeng He` 及 `Sumit Vij`。
+这些发布经理们还要特别感谢测试和发布工作组负责人 Eric Van Norman （IBM）在整个发布周期中提供的帮助和指导。
```
Comment (windsonsea) [inline content/zh/news/releases/1.21.x/announcing-1.21/_index.md:17]:
```suggestion
这些发布经理们特别感谢测试和发布工作组负责人 Eric Van Norman（IBM）在整个发布周期中所提供的帮助和指导。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/_index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## 新特性 {#whats-new}
+
+### 通过兼容性版本轻松升级 {#easing-upgrades-with-compatibility-versions}
+
+Istio 1.21 引入了一个被称为[兼容性版本](/zh/docs/setup/additional-setup/compatibility-versions/)的新概念。
+
+兼容性版本解决了一项 Istio 中长期存在的问题：
+随着时间的推移，对于错误修复、改进与生态系统其他部分的集成、
+提高安全性或修复非预期行为，可能都需要对 Istio 的行为做出变更。
+然而，即使是最小的行为变化也可能会导致像 Istio 这样在数千家公司生产环境中部署的项目的升级问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.21.x/announcing-1.21/_index.md:36]:
```suggestion
然而，即使是最小的行为变更也可能会导致像 Istio 这样在数千家公司生产环境中部署的项目出现升级问题。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/_index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Istio 1.21.0 已得到 Kubernetes `1.26` 到 `1.29` 的官方正式支持。
+{{< /tip >}}
+
+## 新特性 {#whats-new}
+
+### 通过兼容性版本轻松升级 {#easing-upgrades-with-compatibility-versions}
+
+Istio 1.21 引入了一个被称为[兼容性版本](/zh/docs/setup/additional-setup/compatibility-versions/)的新概念。
+
+兼容性版本解决了一项 Istio 中长期存在的问题：
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/_index.md:33]:
```suggestion
兼容性版本解决了 Istio 长期以来的一个问题：
```

**Review by windsonsea — COMMENTED** [review]
Good work! Here are some text suggestions

File: content/zh/news/releases/1.21.x/announcing-1.21/_index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+随着时间的推移，对于错误修复、改进与生态系统其他部分的集成、
+提高安全性或修复非预期行为，可能都需要对 Istio 的行为做出变更。
+然而，即使是最小的行为变化也可能会导致像 Istio 这样在数千家公司生产环境中部署的项目的升级问题。
+最好的情况是，这仅仅使得升级更具挑战性 - 而最坏的情况是，它会迫使用户根本无法升级！
```
[the remark this answers is not in the document]
Comment (hanxiaop) [reply content/zh/news/releases/1.21.x/announcing-1.21/_index.md:37]: I'd say "无法" is not very accurate here:
```suggestion
最好的情况是，这仅仅使得升级更具挑战性；而最坏的情况是，它促使使用户根本不去升级！
```

**Comment by hanxiaop** [comment]
@windsonsea It looks like we are reviewing at the same time. I can do a double check once your review is finished.

File: content/zh/news/releases/1.21.x/_index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: 1.21.x Releases
+description: Announcements for the 1.21 release and its associated patch releases.
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply content/zh/news/releases/1.21.x/_index.md]: Done

**Comment by windsonsea** [comment]
> @windsonsea It looks like we are reviewing at the same time. I can do a double check once your review is finished.

lol, it's your turn now.

**Comment by hanxiaop** [comment]
> > @windsonsea It looks like we are reviewing at the same time. I can do a double check once your review is finished.
> 
> lol, it's your turn now.

Ok I will take a look.

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+- **改进** 改进了变量 `PILOT_MAX_REQUESTS_PER_SECOND`（限制传入请求的速率，之前默认为 25.0）
+  和 `PILOT_PUSH_THROTTLE`（限制并发响应数量，之前默认为 100），
+  如果没有明确配置则自动随 istiod 所进行的 CPU 大小进行扩展。
+
+- **新增** 添加了在各种防火墙规则中配置 `istio-iptables` 使用 IPv4 环回 CIDR 的功能。
+  ([Issue #47211](https://github.com/istio/istio/issues/47211))
+
+- **新增** 添加了当设置网络拓扑之前将工作负载添加到 Ambient 网格，自动设置工作负载默认网络的支持。
+  之前，当您在 Istio 根命名空间上设置 `topology.istio.io/network` 时，
+  您需要手动部署 Ambient 工作负载以使网络变更生效。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:29]:
```suggestion
  您需要手动重新部署 Ambient 工作负载以使网络变更生效。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+  现在，即使 Ambient 工作负载没有网络标签，网络也会自动被更新。
+  请注意，如果您的 ztunnel 与您在 Istio 根命名空间的 `topology.istio.io/network`
+  标签中设置的网络不在同一网络中，则您的 Ambient 工作负载将无法相互通信。
+
+- **新增** 添加了网关部署控制器上的命名空间发现选择器支持。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:34]:
```suggestion
- **新增** 添加了网关部署控制器上的命名空间发现选择器 `discoverySelectors` 支持。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+
+- **修复** 修复了 `pilot-agent istio-clean-iptables`
+  命令无法清理为 Istio DNS 代理生成的 iptables 规则的问题。
+  ([Issue #47957](https://github.com/istio/istio/issues/47957))
+
+- **修复** 修复了在初始 `WorkloadEntry` 创建后不久发生自动注册和清理时，
+  自动注册 `WorkloadEntry` 资源清理缓慢的问题。
+  ([Issue #44640](https://github.com/istio/istio/issues/44640))
+
+- **修复** 修复了 Istio 在扩展时对 `StatefulSets` / 无头`Service` 端点执行额外的 XDS 推送的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:91]:
```suggestion
- **修复** 修复了 Istio 在扩展时对 `StatefulSets` / 无头`Service` 端点执行额外的 xDS 推送的问题。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+- **修复** 修复了启用双栈时未正确创建 QUIC 侦听器的问题。
+  ([Issue #48336](https://github.com/istio/istio/issues/48336))
+
+- **修复** 修复了 `convertToEnvoyFilterWrapper`
+  返回无效补丁的问题，该补丁在应用时可能会导致空指针异常。
+
+- **修复** 修复了更新服务的 `targetPort` 不会触发 xDS 推送的问题。
+  ([Issue #48580](https://github.com/istio/istio/issues/48580))
+
+- **修复** 修复了在没有配置更改时不必要执行集群内分析的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:110]:
```suggestion
- **修复** 修复了在没有配置更改时不必要地执行集群内分析的问题。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+
+- **新增** 添加了在 `EnvoyFilter` 中匹配 `ApplicationProtocols` 的支持。
+
+- **移除** 移除了对 `PodDisruptionBudget` 的 `policy/v1beta1` API 版本的支持。
+
+- **移除** 移除了使用 `BOOTSTRAP_XDS_AGENT` 实验功能在启动时应用 `BOOTSTRAP` `EnvoyFilter` 补丁的功能。
+
+## 安装 {#installation}
+
+- **改进** 如果 Envoy 进程提前终止，改进了优雅终止逻辑。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:206]:
```suggestion
- **改进** 如果 Envoy 进程提前终止，则停止优雅终止逻辑。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 11]
```diff
@@ -0,0 +1,337 @@
...
+
+- **更新** 更新了 Kiali 插件至版本 v1.79.0。
+
+- **新增** 在 Helm Chart 中添加了 Gateway HPA 的可配置扩展行为。
+  ([使用方法](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#configurable-scaling-behavior))
+
+- **新增** 在 Gateway Chart 中添加了 `allocateLoadBalancerNodePorts` 配置选项。
+  ([Issue #48751](https://github.com/istio/istio/issues/48751))
+
+- **新增** 添加了一条消息，指示默认 Webhook 从修订安装转变为默认安装。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:217]:
```suggestion
- **新增** Webhook 从修订安装转变为默认安装时，添加一条消息指示。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 12]
```diff
@@ -0,0 +1,337 @@
...
+  ([Issue #48643](https://github.com/istio/istio/issues/48643))
+
+- **新增** 向 istiod 部署中添加了 `affinity` 字段。该字段用于控制 istiod Pod 的调度。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:220]:
```suggestion
- **新增** 向 Istiod 部署中添加了 `affinity` 字段。该字段用于控制 Istiod Pod 的调度。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 13]
```diff
@@ -0,0 +1,337 @@
...
+
+- **新增** 向 istiod 部署中添加了 `tolerations` 字段。该字段用于控制 istiod Pod 的调度。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:222]:
```suggestion
- **新增** 向 Istiod 部署中添加了 `tolerations` 字段。该字段用于控制 Istiod Pod 的调度。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 14]
```diff
@@ -0,0 +1,337 @@
...
+
+- **新增** 添加了对 Helm 安装“配置文件”的支持。尝试使用 `--set profile=demo`！
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:224]:
```suggestion
- **新增** 添加了对 Helm 安装的 `profiles` 的支持。尝试使用 `--set profile=demo`！
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 15]
```diff
@@ -0,0 +1,337 @@
...
+  ([Issue #47960](https://github.com/istio/istio/issues/47960))
+
+- **修复** 修复了当 Pod 或其自定义所有者的名称超过 63 个字符时注入失败的问题。
+
+- **修复** 修复了导致 Istio CNI 在最小/锁定节点上停止运行的问题（例如没有 `sh` 二进制文件）。
+  新逻辑无需任何外部依赖即可运行，并且如果遇到错误（这可能是由 SELinux 规则等原因引起的），
+  它将尝试继续运行。特别指出，这修复了在 Bottlerocket 节点上运行 Istio 的问题。
+  ([Issue #48746](https://github.com/istio/istio/issues/48746))
+
+- **修复** 修复了 `istio-proxy` 容器的自定义注入在 OpenShift 上不起作用的问题，
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:244]:
```suggestion
- **修复** 修复了在 OpenShift 上自定义注入`istio-proxy` 容器不起作用的问题，
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 16]
```diff
@@ -0,0 +1,337 @@
...
+  由于 OpenShift 对 Pod 的 `SecurityContext.RunAs` 字段设置方式导致。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:245]:
```suggestion
  是由于 OpenShift 对 Pod 的 `SecurityContext.RunAs` 字段设置方式导致。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 17]
```diff
@@ -0,0 +1,337 @@
...
+  因为 `PodSecurityPolicy` 已被[弃用](https://kubernetes.io/zh-cn/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)。
+
+- **移除** 移除了 `istioctl experimental revision` 命令。
+  可以通过稳定的 `istioctl tag list` 命令检查修订。
+
+- **移除** 移除了运行 `istioctl install` 时创建的 `installed-state` `IstioOperator`。
+  之前这仅提供了已安装内容的快照。然而，它是一个常见的混乱来源（因为用户会更改它，但什么也不会发生），
+  并且不能可靠地代表当前状态。由于这些用途不再需要 `IstioOperator`，
+  `istioctl install` 和 `helm install` 不再安装`IstioOperator` CRD。
+  请注意，这仅影响 `istioctl install`，而不影响集群内的 Operator。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:265]:
```suggestion
  请注意，这仅影响 `istioctl install`，而不影响 in-cluster Operator。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 18]
```diff
@@ -0,0 +1,337 @@
...
+  ([Issue #47552](https://github.com/istio/istio/issues/47552))
+
+- **移除** 在安装过程中移除了对 `.Values.cni.psp_cluster_role` 的支持，
+  因为 `PodSecurityPolicy` 已被[弃用](https://kubernetes.io/zh-cn/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)。
+
+- **移除** 移除了 `istioctl experimental revision` 命令。
+  可以通过稳定的 `istioctl tag list` 命令检查修订。
+
+- **移除** 移除了运行 `istioctl install` 时创建的 `installed-state` `IstioOperator`。
+  之前这仅提供了已安装内容的快照。然而，它是一个常见的混乱来源（因为用户会更改它，但什么也不会发生），
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:262]:
```suggestion
  之前这仅提供了已安装内容的快照。然而，它是一个常见的混淆来源（因为用户会更改它，但什么也不会发生），
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 19]
```diff
@@ -0,0 +1,337 @@
...
+  并且不能可靠地代表当前状态。由于这些用途不再需要 `IstioOperator`，
+  `istioctl install` 和 `helm install` 不再安装`IstioOperator` CRD。
+  请注意，这仅影响 `istioctl install`，而不影响集群内的 Operator。
+
+## istioctl
+
+- **改进** 改进了注入器列表以排除 Ambient 命名空间。
+
+- **改进** 通过减少对 k8s API 的调用量，改进了 `bug-report` 性能。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:271]:
```suggestion
- **改进** 通过减少对 k8s API 的调用量，改进了 `istioctl bug-report` 性能。
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 20]
```diff
@@ -0,0 +1,337 @@
...
+  字段而不是 `targetRef` 字段，这将导致资源无效。
+  ([Issue #48273](https://github.com/istio/istio/issues/48273))
+
+- **新增** 添加了消息 IST0167 以警告用户，
+  Sidecar 等策略在应用于 Ambient 命名空间时不会产生任何影响。
+  ([Issue #48105](https://github.com/istio/istio/issues/48105))
+
+- **新增** 添加了引导程序摘要到所有配置转储的摘要中。
+
+- **新增** 添加了一些可以选择 Pod 命令的 Kubernetes Pod 完整功能，
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:294]:
```suggestion
- **新增** 对某些可以选择 pod 的命令的 Kubernetes pod 的自动补全，
```

File: content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md [hunk 21]
```diff
@@ -0,0 +1,337 @@
...
+  资源或资源注解 `proxy.istio.io/config` 设置时，错误报告 IST0158 消息的问题。
+
+- **修复** 修复了 `proxy-config ecds` 未显示所有 `EcdsConfigDump` 的问题。
+
+- **修复** 修复了注入器列表具有为同一注入器 Hook 显示重复命名空间的问题。
+
+- **修复** 修复了 `analyze` 在分析包含集群中已存在资源的文件时无法正常工作的问题。
+  ([Issue #44844](https://github.com/istio/istio/issues/44844))
+
+- **修复** 修复了 `analyze` 报告空文件错误的问题。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21/change-notes/index.md:327]:
```suggestion
- **修复** 修复了当分析的文件中没有内容时 `analyze` 报告错误的问题。
```

**Review by hanxiaop — APPROVED** [review]
LGTM

**Review by windsonsea — APPROVED** [review]
Thanks

**Pull request merged**